src/rgw/rgw_common.h

```diff
@@ -1881,6 +1884,8 @@ struct rgw_obj {
     return in_extra_data;
   }
 
+  const std::string& get_bucket_name() const { return bucket.name; }
```

Comment:
is this used anywhere? exposing this here seems error-prone, because callers are likely to ignore `bucket.tenant`

Reply:
not used anywhere, will remove